docs/configure.rst

```diff
@@ -56,9 +51,12 @@ For the complete listing and description of all configuration options, you shoul
    :start-after: # rfmdocstart: site-configuration
    :end-before: # rfmdocend: site-configuration
 
-There are three required sections that each configuration file must provide: ``systems``, ``environments`` and ``logging``.
+There are three required sections that each configuration must provide: ``systems``, ``environments`` and ``logging``.
```

Comment:
The reason I had removed this completely is because they are not required anymore, from the user's perspective. He can skip any of them and ReFrame won't complain.

Reply:
Yes, I understood this, but essentially they are still required to be present in the final configuration, therefore defined somewhere. That's why I kept it and rephrased it.